metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -507,13 +507,13 @@ class MetalsLanguageServer(
       capabilities.setDocumentSymbolProvider(true)
       capabilities.setDocumentFormattingProvider(true)
       if (initializeParams.supportsCodeActionLiterals) {
-        capabilities.setCodeActionProvider(true)
-      } else {
         capabilities.setCodeActionProvider(
           new CodeActionOptions(
             List(CodeActionKind.QuickFix, CodeActionKind.Refactor).asJava
           )
         )
+      } else {
+        capabilities.setCodeActionProvider(true)
       }
```

Comment:
I got this wrong in #1251. When the client supports code action literals we can send the `CodeActionOptions` version, otherwise we just send `true`